protocol/avdl/notify_paperkey.avdl

```diff
@@ -0,0 +1,8 @@
+
+@namespace("keybase.1")
+protocol NotifyPaperKey {
+  import idl "common.avdl";
+
+  @notify("")
+  void paperKeyCached(UID uid, KID encKID, KID sigKID);
```

Comment:
That's the first I've heard of this.

Nothing in client/protocol/avdl uses `oneway`.  Would you like me to change it?